substratum_tab_light_themes.xml

```diff
@@ -1,11 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <substratum>
-    <theme id="Aurora">
```

Comment:
ditto

Reply:
Its not removed from playstore but its old af, there are more themes by him
what should i do?

Reply:
I'll email the themer and see what he has to say.